index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>mobile-view</title>
+    
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>    
+    <main>
+        <header>
+        <div class="wrapper1">
+          <div class="header-wrapper">
+            <div class="logo"><a href="index.html">My Logo</a></div>
+            <nav>
+              <a href="">
+                <img id="menu1" src="images/menu1.png" alt="menu" />
```

Comment:
- [ ] Please add `./` before the file source name because now it's not working on the web like the image below.

![image](https://user-images.githubusercontent.com/68796232/159012483-b882201a-3d76-4541-a535-6bc07deff470.png)